music_assistant/models/player.py

```diff
@@ -829,6 +828,12 @@ def active_source(self) -> str | None:
         # in case player's source is None, return the player_id (to indicate MA is active source)
         return self._active_source or self.player_id
 
+    @active_source.setter
+    @final
+    def active_source(self, source: str) -> None:
+        """Set the active source of the player."""
+        self._attr_active_source = source
+
```

Comment:
We cant assume each player implementation uses the properties this way. So this setter is not accepted

Reply:
Oh of course, for players with internal sources like AVRs etc.  I can get a bit myopic when thinking from a squeezelite perspective!